tests/test_workers/test_scm.py

```diff
@@ -465,20 +465,22 @@ def test_fetch_source_invalid_archive_exists(mock_clone, mock_verify, caplog, gi
 
 
 @mock.patch("git.Repo")
-def test_update_git_submodules(mock_repo):
+@mock.patch("cachito.workers.scm.run_cmd")
+def test_update_git_submodules(mock_git_submodule_update, mock_repo):
```

Comment:
Should we use this opportunity and add type annotations to this function signature here? If so, also look at the `test_update_git_submodules_failed` function.

Reply:
Added :+1: